system-int-test/test/cmr/system_int_test/ingest/bulk_update/bulk_update_test.clj

```diff
@@ -573,7 +577,7 @@
           ;; Check that each concept was updated
           (doseq [concept-id concept-ids
                   :let [concept (-> (search/find-concepts-umm-json :collection
-                                                                   {:concept-id concept-id})
+                                                                   {:concept-id (first concept-ids)})
```

Comment:
This line should not have been changed.

Reply:
You are correct 😄 - I had changed it for debugging and forgot to undo that